docs/source/notes/multiprocessing.rst

```diff
@@ -9,9 +9,13 @@ memory and will only send a handle to another process.
 
 .. note::
 
-    When a :class:`~torch.Tensor` is sent to another process, both
-    the :attr:`~torch.Tensor` data and :attr:`torch.Tensor.grad` are going to be
-    shared.
+    When a :class:`~torch.Tensor` is sent to another process, the
+    :class:`~torch.Tensor` data is shared. If :attr:`torch.Tensor.grad` is
+    not ``None``, it is also shared. After a :class:`~torch.Tensor` without
+    a :attr:`torch.Tensor.grad` field is sent to the other process, it
+    creates a standard process-specific ``.grad`` :class:`~torch.Tensor` that
+    is not automatically shared across all processes like how the
```

Comment:
shouldn't the word here be "unlike", rather than "like"? Maybe with a comma before it too.